Source/WebCore/dom/Subscriber.cpp

```diff
@@ -133,17 +132,13 @@ void Subscriber::close(JSC::JSValue reason)
 
 bool Subscriber::isInactiveDocument() const
 {
-    RefPtr document = dynamicDowncast<Document>(scriptExecutionContext());
+    RefPtr document = dynamicDowncast<Document>(protectedScriptExecutionContext());
     return (document && !document->isFullyActive());
 }
 
 void Subscriber::reportErrorObject(JSC::JSValue value)
 {
-    auto* context = scriptExecutionContext();
-    if (!context)
```

Comment:
Why did we drop the null check for the context here?

Reply:
No reason, ill revert. Just tried to keep this consistent to the other PR I've been submitting.